src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/RenderBackground.kt

```diff
@@ -4,136 +4,96 @@ import at.hannibal2.skyhanni.config.core.config.Position
 import at.hannibal2.skyhanni.data.GuiEditManager
 import at.hannibal2.skyhanni.data.GuiEditManager.Companion.getAbsX
 import at.hannibal2.skyhanni.data.GuiEditManager.Companion.getAbsY
-import at.hannibal2.skyhanni.data.GuiEditManager.Companion.getDummySize
 import at.hannibal2.skyhanni.utils.ColorUtils.toChromaColor
 import at.hannibal2.skyhanni.utils.RenderUtils
-import io.github.moulberry.notenoughupdates.util.Utils
+import at.hannibal2.skyhanni.utils.renderables.Renderable
 import net.minecraft.client.Minecraft
 import net.minecraft.client.gui.ScaledResolution
-import net.minecraft.client.renderer.GlStateManager
 import net.minecraft.util.ResourceLocation
-import org.lwjgl.opengl.GL11
 
-class RenderBackground {
+object RenderBackground {
 
-    fun renderBackground() {
-        val alignmentConfig = CustomScoreboard.alignmentConfig
+    private val textureLocation by lazy { ResourceLocation("skyhanni", "scoreboard.png") }
+
+    internal fun addBackground(renderable: Renderable): Renderable {
```

Comment:
This used to cause flickering in the GUIEditor with the old rendering that didnt use renderables
Does it still flicker? Please check

Reply:
It does not flicker